turn/src/allocation/mod.rs

```diff
@@ -31,6 +32,32 @@ const RTP_MTU: usize = 1500;
 
 pub type AllocationMap = Arc<Mutex<HashMap<FiveTuple, Arc<Allocation>>>>;
 
+/// Information about an [`Allocation`].
+#[derive(Debug, Clone)]
+pub struct AllocationInfo {
+    /// [`FiveTuple`] of this [`Allocation`].
+    pub five_tuple: FiveTuple,
+
+    /// Username of this [`Allocation`].
+    pub username: String,
+
+    /// Relayed bytes with this [`Allocation`].
+    ///
+    /// If it is [`None`] it means the relayed bytes are not counted.
+    pub relayed_bytes: Option<usize>,
```

Comment:
As above, is there are reason for this whole struct to exist if the metrics feature is not enabled? If there is I would suggest using

```rust
#[cfg(feature = "metrics")]
pub relayed_bytes: usize
```

instead of an `Option`

Reply:
Generally this struct exists not only for `relayed_bytes`. It also provides information about `five_tuple` and `name` which can be useful for `delete_allocations_by_username` method, for example. By the way, `AllocationInfo` can be extended to bring more information about. And `relayed_bytes` is one of many possible fields.

Reply:
Aight that sounds fair. Instead of Option it would seem that feature gating `relayed_bytes` would be a nicer API both when using the feature and not, WDYT?